src/format.rs

```diff
@@ -3,21 +3,40 @@ use std::io::Write;
 use std::process::Command;
 use std::process::Stdio;
 
+use log::debug;
+
 // TODO: This should not call rustfmt from the commandline.
 // Instead, we should use it as a library. Oh well.
 pub fn format(input: &String) -> Result<String, InspectError> {
-    let mut cmd = Command::new("rustfmt")
+    let mut builder = Command::new("rustfmt");
+    builder
         .arg("--emit")
         .arg("stdout")
         .stdin(Stdio::piped())
-        .stdout(Stdio::piped())
-        .spawn()?;
+        .stderr(Stdio::piped())
+        .stdout(Stdio::piped());
+
+    let mut cmd = builder.spawn()?;
 
     cmd.stdin
         .as_mut()
         .ok_or(InspectError::Rustfmt("Cannot pipe input".to_string()))?
         .write_all(input.as_bytes())?;
 
     let output = cmd.wait_with_output().expect("Failed to read stdout");
+
+    // Only log out formatting errors when the verbose flag is specified, 
```

Comment:
I think this comment could be changed because the `verbose` flag isn't really used anymore in that context. 😃

Reply:
Since this was just a minor change, I went ahead and fixed it real quick. Merging now. :)

Reply:
Ah yes! thanks, I forgot about that comment.